src/vumps_localham.jl

```diff
@@ -167,14 +215,13 @@ function (A::Aᴿ)(x)
   return xT - xR
 end
 
-function right_environment(hᴿ, ψ; tol=1e-15)
+# TODO Generate all environments, why? Only one is needed in the sequential version
```

Comment:
It should be ok to only compute the environments that are needed (in practice there are a few needed, since you need the environments for `AC` and the bond matrices `C` on either side of `AC`). I think this was just leftover from copying code over from the parallel version.

However, in practice only one of the environments is computed using a linear solver (`linsolve`) and the rest are derived from that one through transfer matrices, so it should be relatively cheap to just compute all of them. We could revisit this if it actually leads to a noticeable performance problem.